apps/web/src/state/farms/hooks.ts

```diff
@@ -148,3 +147,18 @@ export const useLpTokenPrice = (symbol: string) => {
 export const usePriceCakeUSD = (): BigNumber => {
   return useCakePriceAsBN()
 }
+
+export const useFarmsV2Data = () => {
+  const { chainId } = useActiveChainId()
+
+  return useSWRImmutable(chainId ? ['useFarmsV2Data', chainId] : null, async () => {
+    const mc = getMasterchefContract(undefined, chainId)
+    const totalRegularAllocPoint = await mc.totalRegularAllocPoint()
+    const cakePerBlock = await mc.cakePerBlock(true)
+
+    return {
+      totalRegularAllocPoint,
+      cakePerBlock,
+    }
+  })
+}
```

Comment:
we shouldn't need another 2 requests

Reply:
I'm not sure what it means...

Reply:
you just need to add `totalRegularAllocPoint` and `cakePerBlock` on `createFarmFetcher.fetchFarms`

Reply:
fixed